libnmstate/validator.py

```diff
@@ -24,7 +24,14 @@
 
 from . import nm
 from . import schema
-from libnmstate.appliers.bond import is_in_mac_restricted_mode
+from libnmstate.appliers.bond import (
+    is_in_mac_restricted_mode,
+    get_bond_slaves_from_state,
+)
```

Comment:
Current code style is not using combined import. Not a must.

Reply:
Fixed!